src/model/Computation.js

```diff
@@ -1,3 +1,5 @@
+/* global console */
+
```

Comment:
i didn't realize this block was down below in the file:

```js
 			if ( console.groupCollapsed ) console.groupCollapsed( '%cshow details', 'color: rgb(82, 140, 224); font-weight: normal; text-decoration: underline;' );
 			const functionBody = prettify( this.signature.getterString );
 			const stack = this.signature.getterUseStack ? '\n\n' + truncateStack( err.stack ) : '';
 			console.error( `${err.name}: ${err.message}\n\n${functionBody}${stack}` );
 			if ( console.groupCollapsed ) console.groupEnd();
```

Since it _is_ there, then I would leave the pragma. though ultimately, that code should move into `log` util.